multi-node/aws/README.md

```diff
@@ -5,24 +5,86 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ## Development
 
+### Download pre-built binary
+
+```sh
+wget https://<binary-url>
+# check checksum
+chmod +x ./kube-aws
+sudo mv kube-aws /usr/bin/
+```
+
 ### Build
 
 Run the `./build` script to compile `kube-aws` locally.
-This depends on having golang available on your workstation.
+
+This depends on having:
+* golang >= 1.5
+
 The compiled binary will be available at `./bin/kube-aws`.
 
-### Custom Kubernetes Manifests
+## Initialize an asset directory
+```sh
+$ mkdir my-cluster
+$ cd ./my-cluster
+$ kube-aws init --cluster-name=my-cluster-name --external-dns-name=my-cluster-endpoint --region=us-west-1 --availability-zone=us-west-1c --key-name=key-pair-name
+```
 
-You may deploy a cluster using a custom CloudFormation template, Kubernetes manifests and install scripts using the `artifactURL` option in your cluster config.
+There will now be a ./cluster.yaml file in the asset directory.
```

Comment:
nit: drop the "./" prefixes. Considering this is for human consumption:

"there will be a `cluster.yaml` file in the asset directory" VS
"there will be a `./cluster.yaml` file in the asset directory"